src/sparsezoo/version.py

```diff
@@ -22,14 +22,17 @@
 
 version_base = "1.7.0"
 is_release = False  # change to True to set the generated version as a release version
+is_dev = False
+dev_number = None
 
 
 def _generate_version():
-    return (
-        version_base
-        if is_release
-        else f"{version_base}.{date.today().strftime('%Y%m%d')}"
-    )
+    if is_release:
+        return version_base
+    elif is_dev:
+        return f"{version_base}.dev{dev_number}"
+    else:
+        return f"{version_base}.{date.today().strftime('%Y%m%d')}"
```

Comment:
This code looks fine bc the sed stuff got merged. Why did we stick with the sed command and not setting env vars for checking is_dev, is_.*?
The problem with using sed is that unless you know that the GHA exists, you wont know when they will be true. 
If using sed, Id add comment saying these variables will be overwritten from GHA

Reply:
sed is setting the values for the flags (i.e setting is_dev, is_version). And then the flags are set here.